workload_generator/src/workload_gen.py

```diff
@@ -13,31 +13,38 @@
 import logging
 import queue
 import time
+#import requests
 from threading import Thread
 
 defaultTestFile = "../workloads/WL_1_USER.txt"
+
 serverAddress   = "localhost"
-serverPort      = 60000
-
-ADD = "ADD"
-QUOTE = "QUOTE"
-BUY = "BUY"
-COMMIT_BUY = "COMMIT_BUY"
-CANCEL_BUY = "CANCEL_BUY"
-SELL = "SELL"
-COMMIT_SELL = "COMMIT_SELL"
-CANCEL_SELL = "CANCEL_SELL"
-SET_BUY_AMOUNT = "SET_BUY_AMOUNT"
-CANCEL_SET_BUY = "CANCEL_SET_BUY"
-SET_BUY_TRIGGER = "SET_BUY_TRIGGER"
-SET_SELL_AMOUNT = "SET_SELL_AMOUNT"
-SET_SELL_TRIGGER = "SET_SELL_TRIGGER"
-CANCEL_SET_SELL = "CANCEL_SET_SELL"
-DUMPLOG = "DUMPLOG"
-DISPLAY_SUMMARY = "DISPLAY_SUMMARY"
+serverPort      = "8000"
```

Comment:
this is where we would specify the port for the server.

lmk dianna if this is correct address/port tuple.

Reply:
Yea this is right, the only change would be to port 80 for the production setup

Reply:
Yea this is right, the only change would be to port 80 for the production setup

Reply:
Yea this is right, the only change would be to port 80 for the production setup

Reply:
doneso! changing serverport to 80.

Reply:
8000 is right if you run the normal docker-compose tho